packages/dso-toolkit/src/components/document-component/document-component.stories-of.ts

```diff
@@ -153,8 +156,13 @@ export function documentComponentStories<Implementation, Templates, TemplateFnRe
             return `https://identifier-eto.overheid.nl/${value}`;
           }
 
+          if (name === "IntIoRef" && attribute === "ref" && element) {
+            return `https://identifier-eto.overheid.nl//join/id/regdata/gm1979/2021/Delfzijlkamerverhuur/nld@2021-08-02;1`;
```

Comment:
Kan deze url niet op dezelfde dynamische manier worden samengesteld zoals op regel 156? Zit in `value` niet hett JOIN ID ? Zo nee, misschien beter de data representatiever maken?